templates/repairs_mechanic.html

```diff
@@ -1 +1,63 @@
-{% include 'layout.html' %}
\ No newline at end of file
+{% include 'layout.html' %}
+
+<div class="corner-title">Umówione naprawy</div>
+
+<!-- add a new part -->
+<div style="background: #f39f32; margin-bottom: 3em; margin-top: 1em;">
```

Comment:
![Zrzut ekranu 2024-03-18 181232](https://github.com/PatrykDu/popcorn-works-fastapi-fullstack-app/assets/113666877/054f0505-ee30-4507-a0b3-f6cafbcf6d16)
Fields should be in line